
Pass -n to php when running tests, to ensure that the system inidir
and hence the installed (old) shared extensions are not loaded.

- when running run-tests.php itself
- passed to run-tests.php to ensure it's passed when running each test case
- in cases where the PHP executable is run by a test case

--- php-5.1.1/ext/xml/tests/bug32001.phpt.tests-dashn
+++ php-5.1.1/ext/xml/tests/bug32001.phpt
@@ -159,7 +159,7 @@
 
 if (XML_SAX_IMPL == 'libxml') {
   $php = getenv('TEST_PHP_EXECUTABLE');
-  preg_match("/^libxml2 Version.*\$/im", `$php -i`, $match);
+  preg_match("/^libxml2 Version.*\$/im", `$php -i -n`, $match);
   echo $match[0], "\n";
 } else {
   echo "libxml2 Version => NONE\n";  
--- php-5.1.1/ext/standard/tests/file/proc_open01.phpt.tests-dashn
+++ php-5.1.1/ext/standard/tests/file/proc_open01.phpt
@@ -9,7 +9,7 @@
 	die("no php executable defined");
 } 
 $proc = proc_open(
-	$php,
+	$php . " -n",
 	array(0 => array('pipe', 'r'), 1 => array('pipe', 'w')),
 	$pipes
 );
--- php-5.1.1/ext/standard/tests/file/bug26938.phpt.tests-dashn
+++ php-5.1.1/ext/standard/tests/file/bug26938.phpt
@@ -5,7 +5,7 @@
 $out = array();
 $status = -1;
 $php = getenv('TEST_PHP_EXECUTABLE');
-exec($php . ' -r \'' 
+exec($php . ' -n -r \'' 
      . '$lengths = array(10,20000,10000,5,10000,3);'
      . 'foreach($lengths as $length) {'
      . '  for($i=0;$i<$length;$i++) print chr(65+$i % 27);'
--- php-5.1.1/ext/standard/tests/file/bug26615.phpt.tests-dashn
+++ php-5.1.1/ext/standard/tests/file/bug26615.phpt
@@ -6,7 +6,7 @@
 <?php
 $out = array();
 $status = -1;
-exec($_ENV['TEST_PHP_EXECUTABLE'].' -r \'for($i=1;$i<=5000;$i++) print "$i\n";\' | tr \'\n\' \' \'', $out, $status);
+exec($_ENV['TEST_PHP_EXECUTABLE'].' -n -r \'for($i=1;$i<=5000;$i++) print "$i\n";\' | tr \'\n\' \' \'', $out, $status);
 print_r($out);
 ?>
 --EXPECT--
